src/loaders/workspace_uploader/workspace_uploader.py

```diff
@@ -788,64 +931,28 @@ def main():
     gfu_service_ver = args.gfu_service_ver
     keep_job_dir = args.keep_job_dir
     catalog_admin = args.as_catalog_admin
-    load_id = args.load_id
-    if not load_id:
-        print("load_id is not provided. Generating a load_id ...")
-        load_id = uuid.uuid4().hex
+    load_id = args.load_id or uuid.uuid4().hex
     print(
         f"load_id is {load_id}.\n"
         f"Please keep using this load version until the load is complete!"
     )
-
     env = args.env
     kb_base_url = loader_common_names.KB_BASE_URL_MAP[env]
 
-    if workspace_id <= 0:
-        parser.error(f"workspace_id needs to be > 0")
-    if batch_size <= 0:
-        parser.error(f"batch_size needs to be > 0")
-    if cbs_max_tasks <= 0:
-        parser.error(f"cbs_max_tasks needs to be > 0")
+    _validate_arguments(workspace_id, batch_size, cbs_max_tasks)
```

Comment:
Hmm, parser.error seems more user friendly than throwing an exception

Reply:
Maybe just pass in the parser?

Reply:
👍

Reply:
If you have the parser you don't need to pass in the other args as well, since they're in the parser, right?